### PR TITLE
Update search suggestions to match updated mocks

### PR DESCRIPTION
**Pull request by lukaw3d**
| Before | After |
| --- | --- |
| lw-oasis-explorer-frontend pages dev_ (2)  |  localhost_1234_ (1) |

### CODE REVIEW

File: src/app/components/Search/index.tsx [hunk 1]
```diff
@@ -28,6 +28,11 @@ const SearchForm = styled('form', {
   // Approximate size of TextField without helperText, so helperText floats.
   height: '47px',
 
+  // Show helper text when focused or filled.
+  ':not(:hover, :focus-within) :not(.MuiFormHelperText-filled).MuiFormHelperText-root': {
```
Comment (buberdds) [inline src/app/components/Search/index.tsx:32]: This is so hard to maintain, even comment is saying show when css selector is actually hiding element. Can't we just render SearchSuggestions when needed? Is that a case we are trying to accomplish?

```
 helperText={
          (isHovered || isFocused) && (
            <SearchSuggestions
              onClickSuggestion={suggestion => {
                setValue(suggestion)
              }}
            />
          )
        }
``` ?

**Review by buberdds — COMMENTED** [review]
not sure it these are parts of this PR, but
there is a weird black border around input
on mockups we have additional yellow indicator (but I don't understand fully understand when to show what looking at mockups)

File: src/app/components/Search/index.tsx (hunk 1 above)
Comment (lukaw3d) [reply src/app/components/Search/index.tsx:32]: Yeah :/

I'm not happy about making it visible on hover, but without it: focus empty, see suggestions, click suggestions, suggestions disappear and don't work

Comment (lukaw3d) [reply src/app/components/Search/index.tsx:32]: I'll simplify to `value && <SearchSuggestions` and revisit later :D

**Review by buberdds — APPROVED** [review]

**Pull request merged**